### ISSUE TITLE
Algorithm for excluded headers in CORS filtered responses is hard to parse

### ISSUE THREAD

**Issue by jdm** [whatwg/fetch#202]
```
A CORS filtered response is a filtered response whose type is "cors", header list
excludes all headers in internal response's header list, except those whose name
is either one of `Cache-Control`, `Content-Language`, `Content-Type`, `Expires`,
`Last-Modified`, and `Pragma`, and except those whose name is both not a forbidden
response-header name and one of the values resulting from parsing
`Access-Control-Expose-Headers` in internal response's header list. 
```
I believe this translates to "everything is excluded, unless: (one of special list) or ((not forbidden) and (in A-C-E-H))", but I've had to read it carefully a number of times to come to that understanding.

**Comment by annevk** [comment]
Would you prefer an algorithmic description? E.g., something that takes internal response's header list and returns the filtered response's header list.

**Comment by jdm** [comment]
My main concern is that it's a really long sentence that contains several negatives and combinations of conjunctions and dysjunctions. Whatever can address that is fine by me.

**Comment by annevk** [comment]
Thank you.
